__tests__/app.test.js

```diff
@@ -91,4 +109,30 @@ describe("app", () => {
         });
     });
   });
+  describe("/api/articles/:article_id", () => {
+    it("responds with an object", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((response) => {
+          expect(typeof response.body).toEqual("object");
+        });
+    });
+    it("200 GET /api/articles/:article_id - responds with specific article from articles", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .then(({ body }) => {
+          const article = body;
+          expect(article).toMatchObject({
+            author: expect.any(String),
```

Comment:
Here we could assert agains actual values as we know which article is coming back